### PR TITLE
Removed the use of cfchecker in one of the integration tests.

### PR DESCRIPTION
**Pull request by pelson**
We already check that the CDL doesn't change, and running the cfchecker manually it is the kind of due diligence that you would expect when making significant CF metadata changes.

Will mean we no longer need to build cfchecker and cdat-lite.

### CODE REVIEW

File: lib/iris/tests/results/usecases/pp_to_cf_conversion/to_netcdf/cf_checker/12187.b_0.txt [hunk 1]
```diff
@@ -1,65 +0,0 @@
...
-------------------
-
-------------------
-Checking variable: sigma
-------------------
-
-------------------
-Checking variable: tendency_of_upward_air_velocity_due_to_advection
-------------------
-ERROR (3.1): Units are not consistent with those given in the standard_name table.
```
Comment (rhattersley) [inline lib/iris/tests/results/usecases/pp_to_cf_conversion/to_netcdf/cf_checker/12187.b_0.txt]: Perhaps there's not much point running cfchecker anyway if we're ignoring the errors! :see_no_evil:

**Comment by rhattersley** [comment]
> running the cfchecker manually it is the kind of due diligence that you would expect when making significant CF metadata changes

From what I've seen, "expect" is somewhat optimistic. :wink:

> Will mean we no longer need to build cfchecker and cdat-lite.

Is that a big deal?

**Comment by QuLogic** [comment]
> Is that a big deal?

For one, the server holding `cdat-lite` appears to be dead. Also, I don't think they support Python 3, IIRC.

**Comment by marqh** [comment]
I agree with QuLogic and pelson, i think having cfchecker and cdat-lite as requirements to run our tests is concerning

if we can remove that need and retain our test coverage I'm all for it

**Comment by pelson** [comment]
> From what I've seen, "expect" is somewhat optimistic

Agreed. Ideally we would be doing automatic CF compliance checking - but IMHO the tool isn't right for this job. Maybe we should look at what we can do to help with functionality for automatic cf checking...

> For one, the server holding cdat-lite appears to be dead

This is a major problem that I've known about for a few weeks and not one I've got to the bottom of. We need to chase this up...

Is this PR something you are prepared to merge @rhattersley?

**Comment by ajdawson** [comment]
>>For one, the server holding cdat-lite appears to be dead

>This is a major problem that I've known about for a few weeks and not one I've got to the bottom of. We need to chase this up...

FWIW I'm pretty sure cdat-lite is considered unsupported software by CEDA, is that right @stephenpascoe?

**Comment by stephenpascoe** [comment]
I don't work at CEDA any more so can't comment on their support, and I am not supporting it.  However, I expect they still use it internally. @agstephens should be able to comment.

Before I left I was encouraging them to port the cfchecker away from cdat-lite.  It should be straightforward to use python-netcdf4 instead, or IRIS itself.

**Comment by ajdawson** [comment]
>I don't work at CEDA any more

I didn't know that, I wouldn't have troubled you if I did @stephenpascoe !

**Comment by agstephens** [comment]
Hi all,

It is true that we (CEDA) are not actively supporting cdat-lite. However, you should still be able to pick up the most recent version from PyPI. If you cannot please let me know. It worked for me today.

Thanks, Ag

**Comment by pelson** [comment]
> If you cannot please let me know. It worked for me today.

https://pypi.python.org/pypi/cdat-lite points to http://ndg.nerc.ac.uk/dist/, which is dead AFAICS.
Is it in your cache maybe? I guess the most important thing right now is a reliable download source, but it is worth thinking about what impact not having cdat-lite has on the cfchecker too...

Thanks for the update @agstephens.

**Comment by pelson** [comment]
In case anybody really wants it, I've uploaded a copy of the cdat-lite source to https://gist.github.com/pelson/c143a7f235123580f9d0/raw/956116edc3cdde662898e389444ba3807f3b1188/cdat_lite-5.2-1.tar.gz. I got this from my conda build source cache (that goodness for caches)

**Comment by rhattersley** [comment]
> if we can remove that need and retain our test coverage I'm all for it

Given we were [ignoring the errors](https://github.com/SciTools/iris/pull/1750#discussion_r36611052) arguably our test coverage is unchanged. :wink:

I've raised the see-no-evil issue at source: metarelate/metOcean#63

**Pull request merged**